tests/core/ssl/test_ssl.py

```diff
@@ -37,12 +36,7 @@ async def establish_connection(server: ChiaServer, self_hostname: str, ssl_conte
             100,
             30,
         )
-        handshake = await wsc.perform_handshake(server._network_id, protocol_version, dummy_port, NodeType.FULL_NODE)
-        await session.close()
-        return handshake
-    except Exception:
-        await session.close()
-        return False
```

Comment:
this interface changed. It used to not throw when failing. I take it we don't rely on that anywhere, yeah?

Reply:
All the calls on `main` are of the following form.

```python
        connected = await establish_connection(farmer_server, self_hostname, ssl_context)
        assert connected is True
```

Looking below, the `True` assertions are just eliminated as the call will fail due to the raised exception while the `False` assertions have been replaced by `pytest.raises()` expecting a particular exception making the new code actually more picky about asserting the failure.

Assuming I haven't missed something somewhere...